src/panels/config/integrations/integration-panels/ozw/ozw-node-config.ts

```diff
@@ -216,6 +290,38 @@ class OZWNodeConfig extends LitElement {
     }
   }
 
+  private _configValueChanged(ev: CustomEvent) {
+    this._configData = {
+      ...this._configData,
+      [(<OzwHaForm>ev.currentTarget)!.parameter]: {
+        [Object.keys(
+          this._configData[(<OzwHaForm>ev.currentTarget)!.parameter]
+        )[0]]: ev.detail.value[Object.keys(ev.detail.value)[0]],
+      },
```

Comment:
Not completely sure what is going on here, why can't we just take `ev.detail.value`? I think that has the right key and value already?